script.js

```diff
@@ -0,0 +1,21 @@
+// Write your functin here
+const toolBar = document.querySelector('.menu-img');
+const mobileNav = document.querySelector('.mobile-menu');
+const cancelX = document.querySelector('.cancel');
+const navList = document.querySelector('#mobile-nav-bar');
+
+function displayMobileMenu() {
+  if (mobileNav.style.display === 'none') mobileNav.style.display = 'block';
+  else mobileNav.style.display = 'none';
+}
+function hideMobileMenu() {
+  if (mobileNav.style.display === 'block') mobileNav.style.display = 'none';
+  else mobileNav.style.display = 'block';
+}
+function hideMenu() {
+  mobileNav.style.display = 'none';
+}
```

Comment:
- [ ] Please 🙏🏻 adhere to the best practices of JavaScript and use modern syntax for your code because it will make sure that you are following the current industries standards and writing good quality code, right now you are using ES5 javascript code, but please use the **ES-Next** syntax according to which you need to use the `arrow` function. Kindly refactor your code according to latest standards.